static/app/views/issueDetails/streamline/sidebar/sidebar.tsx

```diff
@@ -89,6 +90,10 @@ export default function StreamlinedSidebar({group, event, project}: Props) {
           <MergedIssuesSidebarSection />
         </Fragment>
       )}
+      <Fragment>
+        <StyledBreak />
```

Comment:
this adds an extra line under Merged Issues even if the sidebar section doesn't appear

Reply:
added a check for the `alert_rule_id` field